ArduCopter/ArduCopter.pde

```diff
@@ -937,6 +940,13 @@ static void update_mount()
 
 #if CAMERA == ENABLED
     camera.trigger_pic_cleanup();
+    if(camera_triggered == false && camera._feedback_pin != -1 && check_digital_pin(camera._feedback_pin) == 0){
```

Comment:
In general using "Magic numbers" is not very clear. I would add a define for what -1 and 0 mean (triggered, nottirggered)